Sign_In_Page.py

```diff
@@ -0,0 +1,23 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+import Secure 
+import config
+import time
+
+
+class Sign_In_Page():
+
+    def __init__(self,driver):
+        self.driver = driver
+        driver.get(config.signInPage_url)
```

Comment:
Only the homepage object includes a default get() because sometime the driver passed in is already at the necessary page. Include a keyword bool parameter and run .get() on True.